topics/ecology/tutorials/panoply_ebv/tutorial.md

```diff
@@ -0,0 +1,256 @@
+- Plot EBV cube arrays.
+- Use any of numerous color tables for the scale colorbar
+- Save plots to GIF, JPEG, PNG or TIFF bitmap images or as PDF or PostScript graphics files.
+- Export animations as MOV or AVI videos
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+> <comment-title>Background</comment-title>
+>
+>There are many online services to get climate data, and it is often difficult to know which ones are up-to date and which resources to trust.
+> Different services provide different Application Programming Interfaces (API), use different terminologies, different file formats etc., which make it difficult for new users to master them all.
+> Therefore in this tutorial, we will be focusing on the usage of Climate data in [Network Common data Form](https://en.wikipedia.org/wiki/NetCDF) (netCDF) because it is the most common data format for storing Climate data.
+> We will be using a freely available dataset containing Essential Climate Variables (sea ice area fraction, surface temperature) from [Copernicus Climate Data Store](https://cds.climate.copernicus.eu/#!/home). We will learn to use panoply to visualize the sea ice area fraction over the poles (southern and northern poles) and surface temperatures for two different years (1979 and 2018).
+{:  .comment}
+
+## NetCDF format
+
+[NetCDF](https://en.wikipedia.org/wiki/NetCDF) data format is a binary format and to be able to read or visualize it, we would need to use dedicated software or libraries that can handle this "special" format. It is self-describing and machine-independent data format that supports the creation, access, and sharing of array-oriented scientific data. NetCDF files usually have the extension *.nc* or *.netcdf*.
+
+For climate and forecast data stored in NetCDF format there are (non-mandatory) conventions on metadata ([CF Convention](http://cfconventions.org/)).
+
+In this tutorial, we will be using data from the [Copernicus Climate Data Store](https://zenodo.org/record/3695482/files/era5-land.nc?download=1) and more precisely a [reanalysis ERA5-Land monthly averaged dataset](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-land-monthly-means?tab=overview) for 2019. Data from Copernicus Climate Data Store is freely available but may require [free registration](https://cds.climate.copernicus.eu/user/register?destination=%2Fcdsapp%23!%2Fdataset%2Freanalysis-era5-land-monthly-means%3Ftab%3Doverview) and license agreement.
+
+## Get data
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial. If you are not inspired, you can name it *Panoply* for example...
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the file from [EBV data portal](https://portal.geobon.org/data/upload/1/public/martins_comcom_id1_20220208_v1.nc)
+>
+>    ```
+>    https://portal.geobon.org/data/upload/1/public/martins_comcom_id1_20220208_v1.nc
```

Comment:
how stable are the URLs here? should we add a "or import from Zenodo" instruction below it?

Reply:
Normally this is a worldwide data warehouse with DOI affected so I hope this will preserve this link and so no need to duplicate data maybe ?

Reply:
ok great :+1: